examples/non-monorepo/tsconfig.json

```diff
@@ -7,8 +7,8 @@
     "strict": true,
     "noEmit": true,
     "esModuleInterop": true,
-    "module": "esnext",
-    "moduleResolution": "bundler",
+    "module": "NodeNext",
+    "moduleResolution": "NodeNext",
```

Comment:
Now, I know what you're thinking. "Why is this change being included in this PR?"

The answer is: I have no idea.

Changing the TSConfig in `basic` is causing issues here in `non-monorepo`. It's nonsense. But, sometimes, you just gotta roll with the punches.